src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private int MAX_POSSIBLE = 100;
```

Comment:
```suggestion
    private int MAX_NUMBER = 100;
```

Reply:
to make it constant, it also should be static and final

Reply:
your checkstyle doesn't pass because checker thinks it's wrong case for a field

Reply:
run mvn clean package and fix errors.